src/style.css

```diff
@@ -18,11 +19,13 @@ main {
 }
 
 h1 {
+  color: #1fb0dc;
   font-size: 26px;
   margin: 50px 0 80px 0;
 }
 
 h2 {
+  color: #f2192f;
   font-size: 16px;
 }
 
```

Comment:
- [ ] optional - 😸 its a lovely concept of using monochromatic color but i would strong suggest u use more color to style page to improve the UX design of the page

Reply:
okay, Thank you I will do that.